src/lang/std/engineConnection.ts

```diff
@@ -914,8 +915,9 @@ export class EngineCommandManager {
   callbacksEngineStateConnection: ((state: EngineConnectionState) => void)[] =
     []
 
-  constructor() {
+  constructor(pool?: string) {
     this.engineConnection = undefined
+    this.pool = pool
```

Comment:
Since `pool` is only used on startup and is passed in through `settings` do we need it to be a public class field on `EngineCommandManager`?

I don't think it's a big deal and I could see it being useful to be able to check `engineCommandManager.pool` elsewhere, I just don't see where we'd be using that right now. If you don't think you need it I think you could drop this and https://github.com/KittyCAD/modeling-app/pull/2246/files#diff-b6994b9cccde4a4952069c83026de1070764e27eee95fba9189ee7db8d4f68e7R40-R45 and it would still work fine for you, but I'm not pressed either way.